frontend/src/libs/helpers/get-error-message.ts

```diff
@@ -0,0 +1,9 @@
+const getErrorMessage = (error: unknown): string => {
```

Comment:
let's create a dir for it and add .helper suffix to file name.
`frontend/src/libs/helpers/get-error-message/get-error-message.helper.ts`